lesson3/_homework/task_02/script.js

```diff
@@ -0,0 +1,32 @@
+
+//                    Adapt Academy 2019 lesson 03 - JS introduction | task 02 by Donatas Dereškevičius 
+
+
+const loadJson = () => {
+
+    fetch (`../../users.json`)
+    .then (response => {
+      if (response.ok) {
+        return response.json();
+      } else {
+        throw new Error('Something went wrong ...');
+      }
+    })
+    .then(users => {
+        let output = '<h2>users with salary over 1 k</h2>';
+        output += '<ul>';
+        users.forEach((user) => {
```

Comment:
Instead of building a string here, you could build a users array and then make an output using that array to results innerHTML.